src/data/nat/choose/multinomial.lean

```diff
@@ -87,9 +99,9 @@ by simpa [finset.sum_pair hab, finset.prod_pair hab] using multinomial_spec {a,
 by simp [multinomial_insert_one {b} f (finset.not_mem_singleton.mpr h) h₁]
 
 lemma binomial_succ_succ [decidable_eq α] (h : a ≠ b) :
```

Comment:
Should this name be changed? There is no `binomial` nor `succ` in the statement. I see something similar also below.

Reply:
There are six lemmas with `binomial` in this file, and I think they are intended to establish a naming convention that `binomial` refers to `multinomial {a, b}` with `a ≠ b` (or maybe more generally finsets of cardinality two, but they don't appear in this file).

Reply:
Hi @riccardobrasca Do you think I should do the rename? You other comments have been marked resolved, and the only change in this PR since your last review was [extracting a lemma](https://github.com/leanprover-community/mathlib/pull/16317/commits/342b13f7599141f816386f978fee1ec203e6ff85).

Reply:
Sorry, I forgot about this PR. It `binomial` is intended to be a name convention than it's OK. Maybe we can write it somewhere in the doc (if it's not already present)? Otherwise LGTM, thanks!

Reply:
It turns out `binomial` traces back to the [original code](https://leanprover.zulipchat.com/#narrow/stream/113489-new-members/topic/Multinomial.20coefficients.20definition/near/293924672) wrote by @kmill, so I think at least one maintainer is OK with the names; I just updated the section docstring to state this naming convention. @pimotte could you just reply `bors r+` to get this PR merged? Thanks!

Reply:
Done! Thanks for finishing this up:)